### ISSUE TITLE
API Key for tests?

### ISSUE THREAD

**Issue by jagill** [bitzeche/zencoder-java#1]
Right now, the tests fail on a default installation due to a missing API Key:

```
Caused by: java.lang.IllegalArgumentException: We need an API key to run these tests
  at de.bitzeche.video.transcoding.zencoder.test.ZencoderClientTest.<init>(ZencoderClientTest.java:50)
```

It'd be nice if we could have these work on setup.  The hard way would be to mock up the Zencoder service and not connect to it at all.  As a quicker solution, we could make a test account specifically for this purpose and use that API key?

**Comment by drallgood** [comment]
Agreed...

I've already created an account for testing purposes, but removed the key prior to comitting.

**Comment by drallgood** [comment]
fixed in e3190f14edea4bece0a830c69f9e713bb3c87c8f
